news/charge_change_score.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Changed the default `charge_changes_score` from 0.0 to 0.1 in the gufe bindings to enable connected networks for 
```

Comment:
```suggestion
* Changed the default ``charge_changes_score`` from 0.0 to 0.1 in the gufe bindings to enable connected networks for 
```

rst needs two back ticks for code formatting (otherwise it's italics)